### PR TITLE
OCPQE-18827: Remove tag `gcp-ipi` and `gcp-upi`

### PR DESCRIPTION
**Pull request by liangxia**
Follow up of https://github.com/openshift/release/pull/47461 and https://github.com/openshift/release/pull/47566

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml [hunk 1]
```diff
@@ -1172,7 +1169,7 @@ tests:
     dependency_overrides:
       OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.13
     env:
-      E2E_RUN_TAGS: '@gcp-ipi and not @storage'
+      E2E_RUN_TAGS: not @storage
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1172]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml [hunk 2]
```diff
@@ -1203,7 +1203,7 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
     env:
       BASELINE_CAPABILITY_SET: None
-      E2E_RUN_TAGS: '@gcp-ipi and not @workloads'
+      E2E_RUN_TAGS: not @workloads
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:1206]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml [hunk 3]
```diff
@@ -1217,7 +1217,7 @@ tests:
     env:
       BASELINE_CAPABILITY_SET: None
       COMPUTE_NODE_TYPE: t2a-standard-4
-      E2E_RUN_TAGS: '@gcp-ipi and not @workloads'
+      E2E_RUN_TAGS: not @workloads
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:1220]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml [hunk 4]
```diff
@@ -1232,7 +1232,7 @@ tests:
     dependency_overrides:
       OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.14
     env:
-      E2E_RUN_TAGS: '@gcp-ipi and not @storage'
+      E2E_RUN_TAGS: not @storage
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:1235]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml [hunk 5]
```diff
@@ -1246,7 +1246,7 @@ tests:
...
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:1249]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml [hunk 6]
```diff
@@ -1327,7 +1327,7 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
     env:
       BASELINE_CAPABILITY_SET: None
-      E2E_RUN_TAGS: '@gcp-ipi and not @workloads'
+      E2E_RUN_TAGS: not @workloads
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:1330]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml [hunk 7]
```diff
@@ -1341,7 +1341,7 @@ tests:
     env:
       BASELINE_CAPABILITY_SET: None
       COMPUTE_NODE_TYPE: t2a-standard-4
-      E2E_RUN_TAGS: '@gcp-ipi and not @workloads'
+      E2E_RUN_TAGS: not @workloads
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:1344]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml [hunk 8]
```diff
@@ -1356,7 +1356,7 @@ tests:
     dependency_overrides:
       OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.14
     env:
-      E2E_RUN_TAGS: '@gcp-ipi and not @storage'
+      E2E_RUN_TAGS: not @storage
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:1359]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml [hunk 9]
```diff
@@ -1370,7 +1370,7 @@ tests:
...
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml:1373]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml [hunk 10]
```diff
@@ -1327,7 +1327,7 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
     env:
       BASELINE_CAPABILITY_SET: None
-      E2E_RUN_TAGS: '@gcp-ipi and not @workloads'
+      E2E_RUN_TAGS: not @workloads
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:1330]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml [hunk 11]
```diff
@@ -1341,7 +1341,7 @@ tests:
     env:
       BASELINE_CAPABILITY_SET: None
       COMPUTE_NODE_TYPE: t2a-standard-4
-      E2E_RUN_TAGS: '@gcp-ipi and not @workloads'
+      E2E_RUN_TAGS: not @workloads
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:1344]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml [hunk 12]
```diff
@@ -1356,7 +1356,7 @@ tests:
     dependency_overrides:
       OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.14
     env:
-      E2E_RUN_TAGS: '@gcp-ipi and not @storage'
+      E2E_RUN_TAGS: not @storage
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:1359]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml [hunk 13]
```diff
@@ -1370,7 +1370,7 @@ tests:
...
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml:1373]: missing ''?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml (hunk 1 above)
Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1172]: Nope.   `make update` does not allow `'xxx'` when the first char is not @

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml:1172]: ok

**Pull request merged**